docs/access-control.md

```diff
@@ -1,24 +1,25 @@
 <!--
-
- Copyright (c) 2024 Snowflake Computing Inc.
- 
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
- 
-      http://www.apache.org/licenses/LICENSE-2.0
- 
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
-
+  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
If we are changing the headers, I suspect we should change _all_ the headers?

Reply:
If you'd like to start a PR for that, that be dope :) So far, we've been doing it in pieces like this one https://github.com/apache/polaris/pull/174/files

Reply:
If you'd like to start a PR for that, that be dope :) So far we've been doing it in pieces like so https://github.com/apache/polaris/pull/174/files